### PR TITLE
Styling icon and link hover states

### PR DESCRIPTION
**Pull request by Lukewh**
Fixes https://github.com/juju/juju-gui/issues/2531

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6900/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6901/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6903/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6907/

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss [hunk 1]
```diff
@@ -14,12 +14,15 @@
     border: 1px solid $light-mid-grey;
     transition: background-color .2s;
     background-color: $canvas-background;
+    // scss-lint:disable ImportantRule
+    color: #888 !important;
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: I think there is a variable for this colour. Is it vanillas fault that an `!important` is required here?

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss [hunk 2]
```diff
@@ -14,12 +14,15 @@
...
+    // scss-lint:disable ImportantRule
 
     &:hover {
       background-color: $white;
       // scss-lint:disable ImportantRule
-      color: $ubuntu-orange !important;
-      // scss-lint:disable ImportantRule
+      color: $warm-grey !important;
+      // scss-lint:disable ImportantRule
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: trailing whitespace?

settings: trim whitespace on save ;)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6910/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6911/

**Comment by huwshimi** [comment]
QA try again

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6914/

File: jujugui/static/gui/src/app/assets/css/_header.scss [hunk 3]
```diff
@@ -59,15 +59,15 @@
...
-        color: $warm-grey;
-        opacity: 1;
-        transition: all .3s;
-
-        &:focus,
-        &:active,
-        &:hover {
-          color: $ubuntu-orange;
-        }
+        // color: $dark-warm-grey;
```
Comment (hatched) [inline jujugui/static/gui/src/app/assets/css/_header.scss]: Is this no longer necessary? Can you remove it?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6916/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6917/

**Comment by huwshimi** [comment]
QA OK :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**